### PR TITLE
BXMSDOC-6473 Spring Boot update configuration section

### PR DESCRIPTION
**Pull request by emmurphy1**
[BXMSDOC-6473](https://issues.redhat.com/browse/BXMSDOC-6473)

PAM draft:
http://file.ork.redhat.com/~emmurphy/BXMSDOC-6473-pam

DM draft:
http://file.ork.redhat.com/~emmurphy/BXMSDOC-6473-dm

### CODE REVIEW

File: doc-content/enterprise-only/springboot/bus-app-config-kie-server-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[id='bus-app-config-kie-server-proc_{context}']
+= Configuring the {KIE_SERVER} identity
+
+After you create your Spring Boot project, you can configure {KIE_SERVER} so that it can be easily identified,
```
Comment (michelehaglund) [inline doc-content/enterprise-only/springboot/bus-app-config-kie-server-proc.adoc]: Change comma to a period.

File: doc-content/enterprise-only/springboot/bus-app-async-execution-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+After you create your Spring Boot project, you can use the `jbpm.executor` parameters to enable asynchronous execution.
+
+.Prerequisites
+* You have a Spring Boot business application ZIP file that you created using the http://start.jbpm.org[business application]  website.
+
+.Procedure
+. Extract the ZIP file and navigate to the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service/src/main/resources` folder, where `<BUSINESS-APPLICATION>` is the name of your Spring Boot project.
+. Open the `application.properties` file in a text editor.
+
+. To enable asynchronous execution, set the value of the `jbpm.executor.enabled` parameter to `true`, uncomment the other `jbpm.executor` parameters, and change the values as required, for example:
```
Comment (michelehaglund) [inline doc-content/enterprise-only/springboot/bus-app-async-execution-proc.adoc]: Probably my bad, but I would have written this as "and change the values as required. For example:"

File: doc-content/enterprise-only/springboot/bus-app-con-rest-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
+[id='bus-app-con-rest-proc_{context}']
+= Configuring REST endpoints for Spring Boot applications
+
+After you create your Spring Boot project, you can configure  the host, port, and path for the REST endpoint for your Spring Boot application.
+
+.Prerequisites
+* You have a ZIP file that you created using the http://start.jbpm.org[business application]  website.
+
+.Procedure
+. Unzip the `<business-application>.zip` file and navigate to the `<business-application>/<business-application>-service/src/main/resources` folder.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/springboot/bus-app-con-rest-proc.adoc]: Extract. Never Unzip. Trademark issue.

File: doc-content/enterprise-only/springboot/bus-app-config-kie-server-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+.Prerequisites
+* You have a Spring Boot business application ZIP file that you created using the http://start.jbpm.org[business application]  website.
+
+.Procedure
+. Extract the ZIP file and navigate to the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service/src/main/resources` folder, where `<BUSINESS-APPLICATION>` is the name of your Spring Boot project.
+. Open the `application.properties` file in a text editor.
+. Configure the {KIE_SERVER} parameters as shown in the following example::
```
Comment (michelehaglund) [inline doc-content/enterprise-only/springboot/bus-app-config-kie-server-proc.adoc]: Remove extra ":"

**Review by michelehaglund — APPROVED** [review]
Minor comments added.

File: doc-content/enterprise-only/springboot/bus-app-async-execution-proc.adoc (hunk 2 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/springboot/bus-app-async-execution-proc.adoc]: Yeah, both are wrong, actually. Should be:
as required, as shown in the following example:  (IBM SG)

For what it's worth, I prefer your version.

File: doc-content/enterprise-only/springboot/bus-app-con-rest-proc.adoc (hunk 3 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/springboot/bus-app-con-rest-proc.adoc]: If you look in the path, that content is outdated. It was updated in another then carried to this one when I rebased.

**Pull request merged**